lib/workers/branch/index.ts

```diff
@@ -560,7 +560,10 @@ export async function processBranch(
     logger.debug(
       `There are ${config.errors.length} errors and ${config.warnings.length} warnings`
     );
-    const { prResult: result, pr } = await ensurePr(config);
+    const { prResult: result, pr } = await ensurePr(config, prLimitReached);
```

Comment:
Potential problem: isn't `prLimitReached` coming from the *branch* limit, instead of PR limit?We should make sure our logic is sound here

Reply:
Right, haven't noticed `getConcurrentPrsRemaining` isn't using any `platform` PR functions

Reply:
I'm about to rewrite it with `platform.findPr({ branchName })`, so both will be using `getPrList()` under the hood

Reply:
Both what? We ideally want the branch limits to check branches, and PR limits to check PRs.

Reply:
Okay, now I see (#7076)